internal-enrichment/glimps-malware/docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  connector:
+    build: .
+    container_name: glimps-connector
+    image: glimps-malware:6.6.15
```

```suggestion
    image: opencti/glimps-malware:6.7.3
```
My bad, I forgot the `opencti/`. However we don't leave the build and container_name in the docker-file, it's only for dev purpose